### DISCUSSION TITLE
Installing on Crostini/Chromebook

### DISCUSSION THREAD

**Discussion by jdcrn** [Homebrew/discussions#2681] [Linux] [answered]
Very new to this. Apologies if I say stupid stuff. 

I'm attempting to install homebrew on my chromebook's linux environment, but it's requesting a password. I've typed in the password that I use to get into the chromebook, which i assume is the only password to the device since i'm the only user of the device, but the installer claims it is incorrect. I don't know where to look for a different password that might work, or how to remove a password so that it can install without one. 

Thanks so much in advance.

Version 96.0.4664.111 (Official Build) (64-bit) Chrome OS Version

**Comment by gromgit** [comment, marked as answer]
https://github.com/Homebrew/install/issues/369 will be of interest.

**Reply by jdcrn** [reply]
well it is certainly a relief to know that the issue is not crass UE, thanks!
